### PR TITLE
[JetBrains] Auto-update also the Stable Gateway Plugin

### PR DESCRIPTION
**Pull request by felladrin**
## Description

Auto-update also the Stable Gateway Plugin.

References:
- [GitHub Actions Expressions: contains](https://docs.github.com/en/actions/learn-github-actions/expressions#contains)

## Related Issue(s)

- https://github.com/gitpod-io/gitpod/issues/13654

## How to test

I've already run in, and you can see the results here: https://github.com/gitpod-io/gitpod/actions/runs/3480056231
The action created this PR: https://github.com/gitpod-io/gitpod/pull/14745 (note it didn't target the main branch, as it was a test.)
image

It was tested with the following steps:

1. Accessed https://github.com/gitpod-io/gitpod/actions/workflows/jetbrains-update-plugin-platform.yml
2. Ran the workflow targeting branch `felladrin/auto-update-stable-gateway-plugin`
    image

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/jetbrains-update-plugin-platform.yml
Comment (andreafalzetti) [inline .github/workflows/jetbrains-update-plugin-platform.yml:28]: Will this auto-merge too?

Comment (felladrin) [reply .github/workflows/jetbrains-update-plugin-platform.yml:28]: It will. I think it's ok, considering it's executing the integration tests.

But we can change and set it to don't merge automatically. What do you think?

Comment (andreafalzetti) [reply .github/workflows/jetbrains-update-plugin-platform.yml:28]: I guess the beauty of this current workflow is that it updates EAP and avoids us manually doing any work, we trade of some risk in favour of getting time back. The integration tests only test connection so things can still break and we won't notice.

I am okay if EAP breaks occasionally but then if we break stable it can lead to incidents and time wasted which we could dedicate to test more before merging the PR (which is created by the other wofklow)

Comment (felladrin) [reply .github/workflows/jetbrains-update-plugin-platform.yml:28]: > should we update stable gateway in [this other job](https://github.com/gitpod-io/gitpod/blob/main/.github/workflows/jetbrains-updates.yml)? together with backend and IDEs?

I don't think we need to.

> If we had stronger integration tests, I'd feel less concerned.

Let me set it to not auto-merge the PR of the Stable Gateway then.

Comment (felladrin) [reply .github/workflows/jetbrains-update-plugin-platform.yml:28]: ✅ Now we only auto-approve EAP Gateway Plugin updates.

https://github.com/gitpod-io/gitpod/blob/6c38d1bb45d675a59d9adf46a0fe3d6c47aa766c/.github/workflows/jetbrains-update-plugin-platform-template.yml#L104-L105

**Review by andreafalzetti — APPROVED** [review]
# 🚀

**Pull request merged**
